shared/lib/ArappFile.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs')
```

Comment:
Nitpick on the file name, we generally use lowercase names unless we're exporting a class